plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageChecksPublisher.java

```diff
@@ -457,7 +458,7 @@ private String getFormatDelta(final Baseline baseline, final Metric metric) {
         return delta + getTrendIcon(delta);
     }
 
-    private Stream<Metric> getMetricStream() {
+    public Stream<Metric> getMetricStream() {
```

Comment:
This method is only used internal, no need to increase visibility
```suggestion
    private Stream<Metric> getMetricStream() {
```

Reply:
Complete